### ISSUE TITLE
Store a seed in your Model for generating new numbers

### ISSUE THREAD

**Issue by eeue56** [bolshchikov/elm-minesweeper#1]
In Grid.elm -

``` elm
fillMines : Int -> Int -> Model -> Model
fillMines minesAmount range grid =
  let
    seed = Random.initialSeed Random.maxInt
```

If you don't use a different seed each time, then it'll just generate the same list over and over and over.

Instead, you should add 

```
seed : Random.Seed
```

to your model, updating it with the seed value returns from calling `Random.generate`

**Comment by bolshchikov** [comment]
@eeue56, yes, you are right. I thought of doing it w/ `Signal`s but didn't figured out how to do it properly.
